inc/osvr/ClientKit/ContextC.h

```diff
@@ -49,7 +49,15 @@ OSVR_EXTERN_C_BEGIN
 
 /** @addtogroup ClientKit
     @{
-*/
+    */
+
+/** @brief Ensures the OSVR server process/thread is running. Call once per process.
+ */
+ OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerAutoStart();
+
+/** @brief Ensures any OSVR server-related resources in the client are released. Call once per process.
+ */
+OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerRelease();
```

Comment:
Is there a specific reason the work done in here can't be done in the client context closing?